helpers/_cookies.tsx

```diff
@@ -1,5 +1,44 @@
+import Cookies from 'js-cookie'
+
 export const COOKIE_NAME_LANGUAGE = 'efw_language'
 export const COOKIE_NAME_ADULT_FILTER = 'efw_adult_filter'
 export const COOKIE_NAME_NEW_TAB = 'efw_new_tab'
 export const COOKIE_NAME_COOKIE_CONSENT = 'efw_cookie_consent_accepted'
 export const COOKIE_NAME_SEARCH_COUNT = 'efw_search_count'
+
+type CookieName =
```

Comment:
wouldn't be the same to just write type CookieName = string since cookie's key are always string?

Reply:
No, this way it can only be exactly those strings.

Reply:
:+1: